src/internal.c

```diff
@@ -9113,7 +9126,28 @@ static int DoVerifyCallback(WOLFSSL* ssl, int ret, ProcPeerCertArgs* args)
         else {
             store->store = &ssl->ctx->x509_store;
         }
-    #endif
+
+    #if defined(OPENSSL_EXTRA)
+        if ((store->param = (WOLFSSL_X509_VERIFY_PARAM*)XMALLOC(
```

Comment:
Should a malloc failure really be ignored here?

Reply:
It makes sense to check it.  Done.